Artifacts.toml

```diff
@@ -0,0 +1,9 @@
+[gap-4-11-0]
+git-tree-sha1 = "48e8399d220c782f4c6cf862f8ac0f436a2a6053"
```

Comment:
What is this SHA?

Reply:
That's a SHA1 hash for the entire artifact directory. It is created using the same algorithm `git` uses to create "tree objects", hence the name.